src/main/java/com/shanebeestudios/skbee/elements/objective/type/Types.java

```diff
@@ -61,13 +62,15 @@ public void change(Objective[] what, @Nullable Object[] delta, ChangeMode mode)
                         }
                     }
                 }));
+        }
 
         if (Skript.classExists("org.bukkit.scoreboard.Criteria")) {
```

Comment:
Please merge the if statement below with this one.

```suggestion
        if (Skript.classExists("org.bukkit.scoreboard.Criteria") && Classes.getExactClassInfo(Criteria.class) == null) {
```

Reply:
Sure, I didn't at first to avoid a long line but I was expecting someone to request this change, on it.